leafsmart-front-end/src/components/CityFacts.js

```diff
@@ -0,0 +1,35 @@
+import { React, useState, useEffect } from "react";
+
+const CityFacts = () => {
+  const [city, setCity] = useState([]);
+  useEffect(() => {
+    const getCity = () => {
+      let geonameid = 6324729;
+      // Teleport API
+      const cityFetch = fetch(
```

Comment:
Were we going to use axios instead of fetch?

Reply:
yep, forgot to fix it in the CityFacts